services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie]:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:
+        return (queryset.filter(genres__in=genres_ids)
+                .filter(actors__in=actors_ids))
+    if genres_ids:
+        return queryset.filter(genres__in=genres_ids)
+    if actors_ids:
+        return queryset.filter(actors__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: str) -> Movie:
+    return Movie.objects.get(id=movie_id)

```

Comment:
The `get_movie_by_id` function uses `Movie.objects.get(id=movie_id)`, which will raise a `DoesNotExist` exception if the movie ID does not exist. Consider handling this exception to provide a more graceful error handling mechanism.